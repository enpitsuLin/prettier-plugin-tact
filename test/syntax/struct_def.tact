struct Point {
    x: Int as int64;
    y: Int as int64;
}

struct Line {
    start: Point;
    end: Point;
}
