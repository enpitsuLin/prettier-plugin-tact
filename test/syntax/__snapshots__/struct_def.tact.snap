struct Point {
    x: Int as int64;
    y: Int as int64;
}

struct Line {
    start: Point;
    end: Point;
}

struct Params {
    name: String = "Satoshi";// default value
    age: Int?;// optional field
    point: Point;// nested Structs
}